doc/faq.rst

```diff
@@ -137,17 +137,15 @@ Using Spyder
 
 .. dropdown:: Q: How do I get Spyder to work with my existing Python packages/environment?
 
-   To work with an existing environment in Spyder, change Spyder’s default Python interpreter.
-   To do so, first go to your terminal (or Anaconda Prompt on Windows), type ``conda info --envs``, and copy the path from the environment you created to your clipboard.
+   To work with an existing environment in Spyder, change Spyder’s default Python interpreter. To do so, in the status bar click the name of current environment and then click :guilabel:`Change default environment in Preferences`.
 
-   .. image:: images/faq/faq-conda-info.png
-      :alt: Navigator showing installing specific version of Spyder
+   .. image:: images/faq/faq-change-environment.png
+      :alt: Change default environment in Preferences option in status bar
 
-   Now, go to :guilabel:`Preferences` in Spyder’s main window, click :guilabel:`Python interpreter` and check :guilabel:`Use the following Python interpreter`.
-   Paste the path you copied before, add ``/bin/python`` at the end on Mac and Linux or ``/python.exe`` on Windows, and click :guilabel:`OK`.
+   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section. Here, check the option :guilabel:`Use the following Python interpreter`. Click the dropdown next to the box below and select your prefered environment.
```

Comment:
```suggestion
   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section.
   Here, select the option :guilabel:`Use the following Python interpreter, and use the dropdown below to select your preferred environment.
```

Fix more line breaks and more typos, and simplify the last two sentences to avoid choppiness.